tools/SourceKit/cmake/modules/AddSwiftSourceKit.cmake

```diff
@@ -18,7 +18,7 @@ function(add_sourcekit_default_compiler_flags target)
     -fblocks)
 endfunction()
 
-function(add_sourcekitd_swifrt_linking target path HAS_SWIFT_MODULES)
+function(add_sourcekitd_swift_rt_linking target path HAS_SWIFT_MODULES)
```

Comment:
@rintaro Should this be `add_sourcekitd_swift_rt_linking` or `add_sourcekitd_swift_linking`?

Reply:
This function corresponds to `_add_swift_runtime_link_flags` in `cmake/modules/AddSwift.cmake`.
@jsoref Since other functions in this file uses `sourcekit` instead of `sourcekitd`, could you rename it to `add_sourcekit_swift_runtime_link_flags`

Reply:
@rintaro: done in https://github.com/apple/swift/pull/42545/commits/18554720c543999f97c24ab0cfb532834efd3c24